### ISSUE TITLE
Separate Guide Tab 

### ISSUE THREAD

**Issue by petergault** [hackerunion-archives/hackerunion.org#32]
I think we should create a separate tab for guides. The guide tab is similar to the members tab, except that it only lists people flagged as guides. 

There would be two lists - a list of available guides, and a list of unavailable guides. Available guides are mentors who are available to mentor more people. Unavailable guides are mentors who already have mentees and are not looking to mentor other people. 

Mentors could set their status as available or unavailable through the Guide Tools button on the home page.

**Comment by mathisonian** [comment]
It's definitely useful to be able to see a list of the guides and their availability. This is already supported on the members page by selecting  "show only guides", perhaps we could just figure out a way to make this more obvious? 

I like the idea of having available / unavailable on their badges. We should make sure that #30 is all ironed out, and we know exactly what guides are before we add this on.

**Comment by petergault** [comment]
Yeah the idea of the separate tab is to make it more prominent. Since guides are a core part of the hacker union, I think it makes sense to create a separate tab to clearly state - here are the available mentors.

We could make it more prominent within the members tab, but I think it is important to state that guides are not just members, but they are also people who are willing to mentor you. Creating a separate tab communicates that separate function. 

I agree that #30 should be figured out first, but I think this goes hand in hand with figuring out how the moderators and guides fit into the overall experience.
